src/window.rs

```diff
@@ -1310,6 +1315,15 @@ impl Default for CursorIcon {
     }
 }
 
+#[derive(Clone)]
+pub struct CursorRgba {
+    pub xhot: u32,
+    pub yhot: u32,
+    pub width: u32,
+    pub height: u32,
+    pub data: Vec<u32>,
```

Comment:
Yes this is ugly, I must clean this part.

Use a crate to avoid to redefine a custom struct could be a good idea. I think `image` is a big one, `rgb` could be a better choice. However the crate to use should be decided by the maintainers.